### PR TITLE
OSDOCS-3640: add note for large disks

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OSDOCS-3640

Link to docs preview: 

- Platform agnostic: https://deploy-preview-45602--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-user-infra-machines-advanced_disk_installing-platform-agnostic

- vSphere: https://deploy-preview-45602--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-disk-partitioning_installing-vsphere

- VMC: https://deploy-preview-45602--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-disk-partitioning_installing-vmc-user-infra

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@miabbott PTAL when you get a chance, this PR adds a couple of `IMPORTANT` notes for using a separate `/var` partition when disk sizes exceed 100G.

The changes in `modules/installation-disk-partitioning.adoc` (this file is included in UPI docs for vSphere and VMC) and `modules/installation-user-infra-machines-advanced.adoc` are identical, but for legacy reasons the docs in the latter aren't modularized so they both need to be updated to remain in sync.

**Comment by jldohmann** [comment]
@cgwalters also PTAL as well. My thought was some small notes, but lmk if something more substantial is needed here.

**Review by cgwalters — APPROVED** [review]

**Review by miabbott — APPROVED** [review]
LGTM!

**Comment by jldohmann** [comment]
c6931ed..28eac81 change italics to double quotes per ISG

File: modules/installation-disk-partitioning.adoc [hunk 1]
```diff
@@ -24,6 +24,11 @@ officially supported for making `/var` or a subdirectory of `/var`, such as `/va
 +
 [IMPORTANT]
 ====
+For disk sizes larger than 100G, and especially disk sizes larger than 1TB, it is strongly recommended to create a separate `/var` partition. See "Creating a separate `/var` partition" for more information.
```
Comment (nalhadef) [inline modules/installation-disk-partitioning.adoc:27]: - Should "100G" be "100GB?"
- Shouldn't recommend thing: https://redhat-documentation.github.io/supplementary-style-guide/#_r
image

File: modules/installation-disk-partitioning.adoc [hunk 2]
```diff
@@ -24,6 +24,11 @@ officially supported for making `/var` or a subdirectory of `/var`, such as `/va
...
+====
++
+[IMPORTANT]
+====
 Kubernetes supports only two filesystem partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
```
Comment (nalhadef) [inline modules/installation-disk-partitioning.adoc:32]: "Filesystem" is two words.
[
](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f)image

File: modules/installation-disk-partitioning.adoc [hunk 3]
```diff
@@ -41,6 +46,11 @@ For example:
 as more images and containers are added to a system.
 * `/var/lib/etcd`: Holds data that you might want to keep separate for purposes such as performance optimization of etcd storage.
 * `/var`: Holds data that you might want to keep separate for purposes such as auditing.
++
+[IMPORTANT]
+====
+For disk sizes larger than 100G, and especially larger than 1TB, it is strongly recommended to create a separate `/var` partition.
```
Comment (nalhadef) [inline modules/installation-disk-partitioning.adoc:52]: - "GB"
- Recommend not recommending. :-)

File: modules/installation-user-infra-machines-advanced.adoc [hunk 4]
```diff
@@ -101,6 +106,11 @@ For example:
 as more images and containers are added to a system.
 * `/var/lib/etcd`: Holds data that you might want to keep separate for purposes such as performance optimization of etcd storage.
 * `/var`: Holds data that you might want to keep separate for purposes such as auditing.
++
+[IMPORTANT]
+====
+For disk sizes larger than 100G, and especially larger than 1TB, it is strongly recommended to create a separate `/var` partition.
```
Comment (nalhadef) [inline modules/installation-user-infra-machines-advanced.adoc:112]: See previous

File: modules/installation-disk-partitioning.adoc (hunk 1 above)
Comment (jldohmann) [reply modules/installation-disk-partitioning.adoc:27]: @nalhadef good catch! What do you think about just omitting that phrase? so something like:

```suggestion
For disk sizes larger than 100G, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" for more information.
```

Comment (nalhadef) [reply modules/installation-disk-partitioning.adoc:27]: Thanks, @jldohmann . Yep, perfect solution!!!

File: modules/installation-disk-partitioning.adoc (hunk 2 above)
Comment (jldohmann) [reply modules/installation-disk-partitioning.adoc:32]: I can change this too, though it's technically outside the scope of my PR changes 😄

**Pull request merged**